docs/content/configuration/global-configuration/configmap-resource.md

```diff
@@ -113,6 +113,7 @@ For more information, view the [VirtualServer and VirtualServerRoute resources](
 |ConfigMap Key | Description | Default | Example |
 | ---| ---| ---| --- |
 |*error-log-level* | Sets the global [error log level](https://nginx.org/en/docs/ngx_core_module.html#error_log) for NGINX. | *notice* |  |
+|*access-log* | Sets the directive [access log](https://nginx.org/en/docs/http/ngx_http_log_module.html#access_log). Only syslog is allowed. | ``/dev/stdout main`` | ``syslog:server=localhost:514`` |
```

Comment:
> Only syslog is allowed

Or standard out also

Reply:
You mean documented? Stdout is the default as it is now

Reply:
```suggestion
|*access-log* | Sets the directive [access log](https://nginx.org/en/docs/http/ngx_http_log_module.html#access_log). only allowed values are standard out (default) and a syslog destination. | ``/dev/stdout main`` | ``syslog:server=localhost:514`` |
```